### PR TITLE
New Fan Type

### PR DESCRIPTION
**Pull request by Ciaran-Oflannagain**
@kevin-hereworks

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/FAN.yaml [hunk 1]
```diff
@@ -18,6 +18,15 @@
...
 ### FAN RUN CONTROL ###
 
+FAN_SS_VSC_WDPM:
+  description: "Fan with start/stop, open-loop speed control and differential pressure monitoring."
+  is_canonical: true
+  implements:
+  - FAN
+  - SS
+  - VSC
+  - WDPM
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FAN.yaml:28]: This is typically discouraged without more context. Differential pressure across what? The zone/room to the outside?

Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/FAN.yaml:28]: Hi Claire - apologies for delay only getting info back now, differential pressure is only monitoring across the fan

File: ontology/yaml/resources/HVAC/entity_types/FAN.yaml [hunk 2]
```diff
@@ -18,6 +18,15 @@
 
 ### FAN RUN CONTROL ###
 
+FAN_SS_VSC_WDPM:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FAN.yaml:21]: please add new GUID

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/FAN.yaml (hunk 2 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/FAN.yaml:21]: @cstirdivant - thanks for response - have added guid. Thanks, Ciaran

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**